superchain/configs/configs.toml

```diff
@@ -4,16 +4,16 @@
 
 [[superchains]]
```

Comment:
This file is only currently used by the rust code. Could we remove this entire file (and the codegen code that creates it) if you prefer for the file to be written to the rust dir? Or do you think this file will be useful in the future for other languages to create their own bindings?

Reply:
Ah that's a great idea actually, will do.